mava/evaluator.py

```diff
@@ -280,7 +280,7 @@ def evaluator_fn(
 
 def make_eval_fns(
     eval_env: Environment,
-    network: Any,
+    network_apply_fn: Any,
```

Comment:
Typing this would be nice :smile: 
You may need to import Callable

```suggestion
    network_apply_fn: Callable[[FrozenDict, Observation], Distribution],
```

Reply:
Typing would be nice, only problem is that at that level it can legally be a RecActorApply or ActorApply function, so it can never be more specific than a generalisation of those. What do you think of assigning it just "Callable" ?

Reply:
Ah lets do `Union[ActorApply, RecActorApply]`